website/src/content/docs/protocol/models/opp-base.mdx

```diff
@@ -193,6 +193,77 @@ model OpportunityBase {
   // Spreads the fields from the Metadata model into the Opportunity model
   ...SystemMetadata;
 }
+```
+
+  </TabItem>
+
+  <TabItem label="Python">
+
+The Python code for this model is:
+
+```python
+from datetime import date, datetime, time
+from uuid import UUID
+from common_grants_sdk.schemas.fields import Event, Money, CustomField, CustomFieldType
+from common_grants_sdk.schemas.models.opp_base import OpportunityBase
+from common_grants_sdk.schemas.models.opp_funding import OppFunding
+from common_grants_sdk.schemas.models.opp_status import OppStatus, OppStatusOptions
+from common_grants_sdk.schemas.models.opp_timeline import OppTimeline
```

Comment:
Small nit, but part of the value of importing these models in the `common_grants_sdk/schemas/__init__.py` file is so that this can become a single `from common_grants_sdk.schemas import ...` or at least just two imports:
- `from common_grants_sdk.schemas.models  import...`
- `from common_grants_sdk.schemas.models  import...`

The benefit of that pattern (besides making the import lines less cluttered) is that consumers of the library API don't have to worry about how the classes are organized within the sub-packages, for example if we wind up putting `OpportunityBase` and `OppFunding` into a new sub-module for opportunities after adding models for `Organization`, `Person`, `Application` etc.